### PR TITLE
Console output cleanup

### PR DESCRIPTION
**Pull request by sharifelgamal**
- Removed all non-templated output functions: OutStyle, ErrStyle, Failure, Fatal, Warning, Success, Usage, WithCode

- Moved package console to out for brevity, renamed: console.OutT -> out.T, console.OutLn -> out.Ln, console.Out -> out.String, console.Arg -> out.V.

### CODE REVIEW

File: cmd/minikube/cmd/ip.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ var ipCmd = &cobra.Command{
 		if err != nil {
 			exit.WithError("Error getting IP", err)
 		}
-		console.Out(ip)
+		out.Out(ip)
```
Comment (tstromberg) [inline cmd/minikube/cmd/ip.go]: Not your fault, but while you are here, this should actually be `out.Ln`. We accidentally broke compatibility with the translation: something I noticed when refactoring TestRegistry.

File: cmd/minikube/cmd/config/prompt.go [hunk 2]
```diff
@@ -36,7 +36,7 @@ func AskForYesNoConfirmation(s string, posResponses, negResponses []string) bool
 	reader := bufio.NewReader(os.Stdin)
 
 	for {
-		console.Out("%s [y/n]: ", s)
+		out.Out("%s [y/n]: ", s)
```
Comment (tstromberg) [inline cmd/minikube/cmd/config/prompt.go]: Find a stutter-free replacement name for out.Out. Perhaps something likue `out.F` or `out.String`?

File: cmd/minikube/cmd/ip.go (hunk 1 above)
Comment (sharifelgamal) [reply cmd/minikube/cmd/ip.go]: Fixed.

File: cmd/minikube/cmd/config/prompt.go (hunk 2 above)
Comment (sharifelgamal) [reply cmd/minikube/cmd/config/prompt.go]: yeah makes sense. I prefer String, since it's not a widely called function.

**Comment by sharifelgamal** [comment]
@minikube-bot OK to test

**Pull request merged**